api/v2/Routes/students/progressquerystring/index.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * @deprecated This file is deprecated. It contains logic to produce a query string for CM, which is no longer used.
```

Comment:
Please add this attribute to all functions from this file you are deprecating so they are marked by intellisense.

Reply:
Will do!